package.json

```diff
@@ -1,21 +1,23 @@
 {
   "name": "pretender",
-  "version": "2.1.0",
+  "version": "3.1.0",
```

Comment:
Took the liberty of bumping the version number since this is a fairly large change to the code base.